lib/data/wrapper.js

```diff
@@ -96,7 +96,7 @@ const data = {
             });
     },
 
-    get: (objectGetInfo, log, cb) => {
+    get: (objectGetInfo, log, params, cb) => {
```

Comment:
I haven't digged into the topic, but on top of head, couldn't those params be part of `objectGetInfo`? Mystique wrapper would extract the relevant params from it and pass them as separate params info then to the Mystique front-end. Unless they can conflict with existing params from `objectGetInfo`, merging them could make the API semantic slightly simpler and avoid the change.